tests/integration/sriov_test.py

```diff
@@ -61,6 +61,47 @@ def test_sriov_decrease_vfs(sriov_interface):
     assertlib.assert_state(sriov_interface)
 
 
+@pytest.mark.xfail(
```

Comment:
These tests are passing on HW with SRIOV?

Reply:
Yes, when eth1 have SR-IOV support. Then it is passing.